src/common/components/markdownViewer/render/markdownRender.ts

```diff
@@ -0,0 +1,51 @@
+import * as MdrObject from 'markdown-it';
+import { MarkdownIt as Mdr } from 'markdown-it';
+import { loadCustomRules } from './markdownRenderRules';
+import { MdrOptions, defaultOptions, codeHighlighter } from './markdownRenderOptions';
+import { MdrCodeStyle, loadCodeStyle, defaultCodeStyle } from './markdownRenderCodeStyle';
+// Keep old import statements, otherwise it won't work with enzyme.
+const mdrPluginAbbr = require('markdown-it-abbr');
```

Comment:
Maybe it's because you need to add these libs to [`vendor`](https://github.com/Lemoncode/LeanMood/pull/168/files#diff-73daf61abe43f39a8ff85a04c77e9d9cR15) bundle.

Reply:
I will do the test and let you know

Reply:
@fjcalzado did you got a change to test this?

Reply:
@fjcalzado did you got a chance to test it?

Reply:
**Done**. It is working now. [Link to commit](https://github.com/Lemoncode/LeanMood/pull/168/commits/87a6c3a78a48494b23170a32f2c3bb25297cc0c2).